shared/fs/index.js

```diff
@@ -1,30 +1,119 @@
 // @flow
 import * as React from 'react'
-import {globalStyles, globalMargins} from '../styles'
-import {Box, Button, Text, ButtonBar} from '../common-adapters'
-
-type Props = {
-  counter: number,
-  you: ?string,
-  increase: () => void,
-  increase10: () => void,
-}
-
-const Fs = ({counter, you, increase, increase10}: Props) => (
-  <Box style={containerStyle}>
-    <Text type="Header">Hi {you}!</Text>
-    <Text type="BodySemibold">Count: {counter}</Text>
-    <ButtonBar align="flex-start">
-      <Button type="Primary" onClick={increase} label="Up by 1" />
-      <Button type="Primary" onClick={increase10} label="Up by 10" />
-    </ButtonBar>
-  </Box>
-)
+import * as Types from '../constants/types/fs'
+import {globalStyles, globalColors, globalMargins, isMobile} from '../styles'
+import {Box, BackButton, Button, ClickableBox, Icon, List, Text} from '../common-adapters'
+import {type IconType} from '../common-adapters/icon'
+
+const stylesCommonCore = {
+  alignItems: 'center',
+  borderBottomColor: globalColors.black_05,
+  borderBottomWidth: 1,
+  justifyContent: 'center',
+}
 
-const containerStyle = {
+const stylesCommonRow = {
+  ...globalStyles.flexBoxRow,
+  ...stylesCommonCore,
+  minHeight: isMobile ? 64 : 48,
+  padding: 8,
+}
+
+const stylesContainer = {
   ...globalStyles.flexBoxColumn,
+  ...globalStyles.fullHeight,
   flex: 1,
-  padding: globalMargins.xlarge,
 }
 
-export default Fs
+type FolderHeaderProps = {
+  title: string,
+}
+
+type FileRowProps = {
+  path: Types.Path,
+  icon: IconType,
+  showFileData: () => void,
+}
+
+type FolderProps = {
+  name: string,
+  path: Types.Path,
+  visibility: Types.Visibility,
+  items: Array<string>,
+  onViewFolder: (p: Path) => void,
+  onViewFile: (p: Path) => void,
+}
+
+const FolderBoxStyle = {...globalStyles.flexBoxColumn, flex: 1, justifyContent: 'stretch'}
```

Comment:
The naming scheme seems different. Did you mean `stylesFolderBox`?

Reply:
Hah yeah I wrote that first, so I hadn't yet adapted the naming conventions of the codebase.